ldm/simplet2i.py

```diff
@@ -686,3 +708,84 @@ def _log_tokenization(self, text):
         print(f"\nTokens ({usedTokens}):\n{tokenized}\x1b[0m")
         if discarded != "":
             print(f"Tokens Discarded ({totalTokens-usedTokens}):\n{discarded}\x1b[0m")
+
+    def _get_base_noise(self, width:int, height:int, variant_amount:float) -> torch.Tensor:
+        base_x_T = None
+        if variant_amount != 0.0:
+            variant_amount = max(0.0, min(1.0, variant_amount))
+            # base noise is made from whatever our seed currently is
+            base_x_T = torch.randn([1,
+                                    self.latent_channels,
+                                    height // self.downsampling_factor,
+                                    width  // self.downsampling_factor],
+                                    device=self.device)
+        return base_x_T
+
+    def _apply_variation_slerp(self, 
+        width:int, height:int,
+        variant_amount:float, variant_seed:int, 
+        base_x_T:torch.Tensor) -> "[torch.Tensor,int]":
+        x_T = None
+        seed = None
+        if variant_amount != 0.0:
+            variant_amount = max(0.0, min(1.0, variant_amount))
+            # no variant seed specified, generate random noise
+            if variant_seed is None:
+                # TODO refactor seed overall?
+                # for now I use uptime to seed variations, 
+                # otherwise you get the same set of variations 
+                # from a seed provided with -S
+                # as seed_everything is used in prompt2image, 
+                # which "restarts" the sequence of seeds.
+                seed = time.monotonic_ns() % np.iinfo(np.uint32).max
```

Comment:
In my original PR I use a random value instead of the timestamp, which is probably nicer:

```py
random.seed() # reset RNG to an actually random state, so we can get a random seed
seed = random.randrange(0,np.iinfo(np.uint32).max)
```

Reply:
nice! I didn't know that would (was thinking it would be based off the prior seed_everything) but tested here and works great, submitted that modification